app/javascript/dashboard/routes/dashboard/inbox/components/InboxContextMenu.vue

```diff
@@ -0,0 +1,63 @@
+<script setup>
+import { defineProps, defineEmits } from 'vue';
+import MenuItem from './MenuItem.vue';
+
+defineProps({
+  isOpen: {
```

Comment:
Do we need this prop? We already have a v-if on the `InboxCard.vue`

Reply:
Done